src/CLI/clitree/macro/vrf_macro.xml

```diff
@@ -28,12 +28,12 @@ limitations under the License.
         >
         <PARAM
           name="vrf-name"
-          help="Name of VRF (Max size 32, prefixed by Vrf_)"
```

Comment:
VRF name length should be 15 chars, please refer HLD for more information.
https://github.com/Azure/SONiC/pull/435/files

Reply:
Changed.